other/pascal_triangle.py

```diff
@@ -0,0 +1,72 @@
+"""
+wissamfawaz12@gmail.com | github.com/wissamfawaz
+This implementation demonstrates how to generate the
+elements of a Pascal's triangle.
+What is Pascal's triangle?
+- Refer to (https://en.wikipedia.org/wiki/Pascal%27s_triangle)
+for more info about this triangle.
+"""
+
+
+def generate_pascal_triangle(num_rows: int) -> None:
+    """
+    Print Pascal's triangle for different number of rows
+    >>> generate_pascal_triangle(1)
+    [[1]]
+    >>> generate_pascal_triangle(2)
+    [[1], [1, 1]]
+    >>> generate_pascal_triangle(3)
+    [[1], [1, 1], [1, 2, 1]]
+    >>> generate_pascal_triangle(4)
+    [[1], [1, 1], [1, 2, 1], [1, 3, 3, 1]]
+    >>> generate_pascal_triangle(5)
+    [[1], [1, 1], [1, 2, 1], [1, 3, 3, 1], [1, 4, 6, 4, 1]]
+    """
+    triangle: list[list[int]] = []
+    for current_row_idx in range(num_rows):
+        populate_current_row(triangle, current_row_idx)
+    print(triangle)
```

Comment:
You should return the triangle, not print it

Reply:
Sure thing. I will return the triangle instead of printing it.